src/game/minicomputer.cpp

```diff
@@ -883,70 +883,68 @@ static void show_build_screen(Handle<Admiral> adm, int32_t line) {
     if (adm != g.admiral) {
         return;
     }
-    make_mini_screen(
-            kBuildMiniScreen,
-            {
-                    text("BUILD SHIPS", false), text("", true), selectable("", build_ship),
-                    selectable("", build_ship), selectable("", build_ship),
-                    selectable("", build_ship), selectable("", build_ship),
-                    selectable("", build_ship), accept("Build"), cancel("Main Menu"),
-            });
+    const miniScreenLineType (&lines)[] = {
```

Comment:
Could you remove the reference here? i.e.:

    const miniScreenLineType lines[] = {

I’m not sure whether const-ref lifetime extension applies when the const-ref is an array and the temporary is an array element.

Same goes for other instances of this line below.

Reply:
Hmm maybe I need to brush up on my C++, but isn't my code an array of const references? I'll make the change though.

Reply:
I think it’s a ref to an array, but I’m not completely confident about that. `make_mini_screen()` needs to take a ref-to-array, because otherwise the array would decay to a pointer and it wouldn’t be possible to deduce the array size (I think). Anyway, the confusion seems like another reason to make it simpler.

Thanks for the change.